docs/ApplePayButton.md

```diff
@@ -24,13 +24,15 @@ In addition to button's type, you can set button's visual appearance. For iOS an
 ![Apple pay button - styles](https://user-images.githubusercontent.com/829963/40891711-daca8ff8-678a-11e8-89f2-26a0c3dcf9ed.png)
 
 ## Props
-| Prop name    | required | Type        | Default Value |
+| Prop Name    | Required | Type        | Default Value |
 |--------------|----------|-------------|---------------|
 | type         | yes      | ButtonType  |               |
 | style        | yes      | ButtonStyle |               |
 | onPress      | yes      | Function    |               |
 | width        | no       | number      |               |
-| height       | no       | number      | 44            |
+| height       | no       | number      |               |
```

Comment:
Did this default value exist for a reason?

Reply:
I thought about this and couldn't think of any reason this default should be here, or why it should be set to `44` of all things when apple allows a minimum button height of 30. Additionally, having a default height value set could override some of our styling, so removing it and instead adding a default `minHeight` value made more sense to me